model/pytorch/dffml_model_pytorch/utils/utils.py

```diff
@@ -34,6 +33,45 @@ def create_layer(layer_dict):
     return sequential_dict
 
 
+def create_network(network_dict):
+    sequential_network = nn.Sequential()
+    for name, layer in network_dict.items():
+        if isinstance(layer, dict):
+            parameters = {k: v for k, v in layer.items()}
+            layer_type = parameters.pop("layer_type")
+            try:
+                if not isinstance(parameters["in_channels"], int):
+                    layers = []
+                    for module in sequential_network.children():
+                        layers.append(module)
+                    for module in reversed(layers):
```

Comment:
Here if the looking at the children of the `nn.Sequential` from reverse as the `in_channels` will be assigned a value which is just above it